.github/workflows/build_and_test.yml

```diff
@@ -34,6 +32,33 @@ jobs:
       - name: Build
         run: |
           cd ${GITHUB_WORKSPACE}/cmd/node && go build .
+          cd ${GITHUB_WORKSPACE}/cmd/seednode && go build .
           cd ${GITHUB_WORKSPACE}/cmd/keygenerator && go build .
           cd ${GITHUB_WORKSPACE}/cmd/logviewer && go build .
           cd ${GITHUB_WORKSPACE}/cmd/termui && go build .
+
+      # On GitHub, we only run the short tests, and we only run them for some OS/ARCH combinations.
+      - name: Run tests
+        run: |
+          GOOS=$(go env GOOS)
+          GOARCH=$(go env GOARCH)
+          GOPATH=$(go env GOPATH)
+
+          # "libwasmer_darwin_amd64.dylib" was built with an unfortunate identifier (in the past), so we need to fix it:
```

Comment:
Unfortunate. Easily fixable with additional commits in all VM repositories (patching the legacy dylib).